nl-core/StructureDefinitions/nl-core-SOAPReport.xml

```diff
@@ -0,0 +1,92 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-soapreport-v1.3-2024EN" />
+    <uri value="https://zibs.nl/wiki/SOAPReport-v1.3(2024EN)" />
+    <name value="zib SOAPReport-v1.3(2024EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Composition" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-SOAPReport" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Composition">
+      <path value="Composition" />
+      <comment value="Zib SOAPReport is implemented using two profiles: this profile implements the SOAPReport concept of the zib, while &lt;http://nictiz.nl/fhir/StructureDefinition/nl-core-SOAPReport.SOAPLine&gt; represents the SOAPLine concept of the zib which is referenced by `.section.entry`. &#xD;&#xA;The Composition resource SHOULD hold a summary in the `.section.text` elements of the information found in the referenced SOAPReport.SOAPLine profile." />
+      <alias value="nl-core-SOAPReport" />
+    </element>
+    <element id="Composition.subject">
+      <path value="Composition.subject" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Resource" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+    </element>
+    <element id="Composition.encounter">
+      <path value="Composition.encounter" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Encounter" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Encounter" />
+      </type>
+    </element>
+    <element id="Composition.author">
+      <path value="Composition.author" />
+      <type>
+        <code value="Reference" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/pattern-NlCoreHealthProfessionalReference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Practitioner" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Organization" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole" />
+      </type>
+    </element>
+    <element id="Composition.title">
+      <path value="Composition.title" />
+      <definition value="Official human-readable label for the composition. Suggested value is either episode title, or the ICPC display name on the A entry." />
```

Comment:
Why do we change the definition here (was already the case in 2020) without a mapping to an information standard or use case?